src/search/FindInFilesUI.js

```diff
@@ -92,7 +94,7 @@ define(function (require, exports, module) {
                         _findBar.enable(true);
                         _findBar.focusQuery();
                         if (zeroFilesToken === FindInFiles.ZERO_FILES_TO_SEARCH) {
-                            _findBar.showError(StringUtils.format(Strings.FIND_IN_FILES_ZERO_FILES, FindUtils.labelForScope(FindInFiles.searchModel.scope)), true);
+                            _findBar.showError(StringUtils.format(Strings.FIND_IN_FILES_ZERO_FILES, FindUtils.labelForScope(searchScope)), true);
```

Comment:
It seems like this change and lines 88-89 aren't needed anymore?  I'm not sure I understand the comment about closing the old search results clearing the scope -- this code worked before, and nothing in this PR seems to introduce a change that would cause the scope to get cleared...